### ISSUE TITLE
[DOXIA-317] wrong section counting in TOC macro

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-doxia#324]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** opened **[DOXIA-317](https://issues.apache.org/jira/browse/DOXIA-317?redirect=false)** and commented

Eg site-plugin-2.0 generates two different tocs for the following:

```
%{toc|section=1|fromDepth=1|toDepth=1}
%{toc|section=2|fromDepth=1|toDepth=1}

Section 1

* Sub-section 1.1
* Sub-section 1.2

Section 2

* Sub-section 2.1
* Sub-section 2.2
```

The toc for section 1 contains the two sub-sections, while the one for section 2 (correctly) contains only the section.

---

**Affects:** 1.0

**Issue Links:**
- [DOXIA-316](https://issues.apache.org/jira/browse/DOXIA-316) Backward issue: wrong section counting in TOC macro

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

This is fixed in Doxia-1.1, see DOXIA-316, there will be no more 1.0.x release.

**Comment by jira-importer** [comment]
**[Dennis Lundberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dennisl@apache.org)** commented

Using Site Plugin 2.1.1 (doxia 1.1.3) both tocs contain only the section, so works as expected.

Do we really want to make a 1.0.1 release of doxia?
